content-src/components/TopSites/SearchShortcutsForm.jsx

```diff
@@ -0,0 +1,143 @@
+import {actionCreators as ac, actionTypes as at} from "common/Actions.jsm";
+import {FormattedMessage} from "react-intl";
+import React from "react";
+
+class SelectableSearchShortcut extends React.PureComponent {
+  render() {
+    const {shortcut, selected} = this.props;
+    const imageStyle = {backgroundImage: `url("${shortcut.icon}")`};
+    return (
+      <div className="top-site-outer">
```

Comment:
There is a bit of duplication of HTML between this component and the `TopSite` component. But I didn't want to make a mess out of that component right now. I figured this duplication is less risky for now.